microprot/snakemake/Snakefile

```diff
@@ -23,6 +23,11 @@ try:
 except KeyError:
     pass
 
+# create output directories if they don't exist
+for _out_dir_name in ['PDB', 'CM', 'AB', 'log', 'pkg']:
+    _out_dir_path = '%s/%s' % (config['MICROPROT_OUT'], _out_dir_name)
```

Comment:
is it save to assume that separator is always '/' what about windows? I am thinking about WCG machines.